File: codeforces/37.divisibility_problem/divisibility_problem.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  int t;
  unsigned long int a, b;
  unsigned int movements;
  cin >> t;
  unsigned long int min_movements[t];
  for (int i = 0; i < t; i++) {
    cin >> a;
    cin >> b;
    movements = 0;
    if ((a % b) != 0) {
      movements = b - (a % b);
    }
    min_movements[i] = movements;
  }
  for (int i = 0; i < t; i++) {
    cout << min_movements[i] << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/800_points/37.divisibility_problem_[1328A]/divisibility_problem.py
# Author: José Rodolfo (jric2002)
n = int(input())
min_movements = []
for i in range(0, n):
  a, b = [int(number) for number in input().split(" ")]
  movements = 0
  if ((a % b) != 0):
    movements = b - (a % b)
  min_movements.append(movements)
for i in range(0, n):
  print(min_movements[i])